app/routes/chat.js

```diff
@@ -1,4 +1,9 @@
 import Ember from 'ember';
 
 export default Ember.Route.extend({
+  beforeModel: function () {
```

Comment:
so...what do you think about letting someone watch the chat but not being able to participate unless they log in? i'm totally cool with this for now but maybe something we can add in the future? i totally dig it in twitch watching the madness that is their chat despite not being logged in